docs/reference/field-types/array.md

```diff
@@ -71,6 +72,22 @@ You can also control whether each item has a toggle to expand it, or is always d
 
 By default, if `inline: true` is set and there are fewer than three fields, each item is fully expanded. You can change this by explicitly setting the inline option to `inline: { alwaysExpand: false }` or `inline: { alwaysExpand: true }`.
 
+### `whenEmpty`
+
+Works only for inline arrays.
```

Comment:
Can you add an extra sentence or two here describing what this is used for? What exactly is being checked for whether or not it is empty?